osgp/platform/osgp-adapter-ws-admin/src/main/java/org/opensmartgridplatform/adapter/ws/admin/application/config/PersistenceConfig.java

```diff
@@ -27,6 +26,12 @@
 @PropertySource(value = "file:${osgp/AdapterWsAdmin/config}", ignoreResourceNotFound = true)
 public class PersistenceConfig extends AbstractPersistenceConfig {
 
+    @Override
+    @Bean(destroyMethod = "close")
+    public DataSource getDataSource() {
```

Comment:
In this class you override the `getDataSource()` method, in other classes you've used `dataSource()`,
I would prefer to use the override in the other classes as well, although probably renaming this method in this class to dataSource() is more in line with the naming conventions used for other beans (like `transactionManager()`)